File: C/Algorithms/561.c
#include <stdlib.h>

int cmpi(const void *x, const void *y) {
  int a = *(const int *)x;
  int b = *(const int *)y;

  return (a > b) - (a < b);
}

int arrayPairSum(int *nums, int numsSize) {
  qsort(nums, numsSize, sizeof(int), cmpi);
  unsigned sum = 0;
  while (numsSize--)
    sum += nums[--numsSize];
  return sum;
}

File: Javascript/Algorithms/561.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var arrayPairSum = function(nums) {
    let len = nums.length, sum=0;
    // Thanks for default converting into strings.
    nums.sort((a, b) => a - b)
    while(len--){
        sum += nums[--len];
    }
    return sum;
};
